msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md

```diff
@@ -844,6 +844,16 @@ Congratulations! You've completed the following app configurations in Azure AD r
 
 ---
 
+## Best practices
+
+- Keep the Azure AD app registration restricted to its original purpose of service to service application.
```

Comment:
minor: I'm not sure whether our customer will be confused about what does the "Azure AD app registration" refers to. Do you think we need to add some description to it like `the Azure AD app registration created for Azure Bot service`?